### PR TITLE
Updated skill level selection to have a description of the skill level.

### PR DESCRIPTION
**Pull request by ttvalkea**

### CODE REVIEW

File: src/components/Skills/SkillLevelSelect.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        </b-col>
+      </b-row>
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'SkillsLevelSelect',
```
Comment (rnousia) [inline src/components/Skills/SkillLevelSelect.vue]: typo, should be SkillLevelSelect

File: src/components/EditProfile/EditSkills.vue [hunk 2]
```diff
@@ -116,11 +116,10 @@
...
       hide-footer
     >
-      <b-radio-group
-        v-model="editedSkill.knows"
+      <SkillLevelSelect
         :options="knowsOptions"
-        plain
-        stacked
+        :initial-value="editedSkill.knows"
+        @optionSelected="knowsOptionSelected($event)"
```
Comment (rnousia) [inline src/components/EditProfile/EditSkills.vue]: event names should be kebab-case
https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

File: src/components/Skills/SkillLevelSelect.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    }
+  },
+  data () {
+    return {
+      selectedOption: this.initialValue
+    }
+  },
+  watch: {
+    selectedOption (value) {
+      this.$emit('optionSelected', value)
```
Comment (rnousia) [inline src/components/Skills/SkillLevelSelect.vue]: custom event should be kebab-case

File: src/components/Skills/SkillLevelSelect.vue [hunk 4]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <div
+      v-for="option in options"
+      :key="option.value"
+    >
+      <b-row>
```
Comment (rnousia) [inline src/components/Skills/SkillLevelSelect.vue]: v-for could be in b-row and extra div could be then removed?

File: src/components/Skills/SkillLevelSelect.vue [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            :value="option.value"
+          >
+            {{ `${option.value} - ${option.text}` }}
+          </b-form-radio>
+        </b-col>
+      </b-row>
+      <b-row>
+        <b-col
+          class="description"
+          :class="selectedOption === option.value ? 'selected-description' : ''"
```
Comment (rnousia) [inline src/components/Skills/SkillLevelSelect.vue]: can be shortened to: :class="selectedOption === option.value ? 'description selected-description' : 'description'"

File: src/components/Skills/SkillLevelSelect.vue (hunk 4 above)
Comment (ttvalkea) [reply src/components/Skills/SkillLevelSelect.vue]: There are two b-rows inside the div, so the current structure seems fine.

**Pull request merged**